recipes/devel/m4.yaml

```diff
@@ -2,6 +2,9 @@ inherit: [autotools, patch]
 
 metaEnvironment:
     PKG_VERSION: "1.4.20"
+    PKG_LICENSE: "LicenseRef-m4"
```

Comment:
Buildroot seems to use `GPL-3.0-or-later`. Looking at the source files seems to confirm this. Any reason why the custom license is needed?